server/routes/weather.js

```diff
@@ -0,0 +1,29 @@
+const express = require('express');
+const axios = require('axios');
+const weather = express.Router();
+const { GOOGLE_MAPS_API_KEY, VISUAL_CROSSING_API_KEY } = process.env;
+
+weather.get('/weather', (req, res) => {
+  const userIpAddress = req.clientIp;
```

Comment:
request IP middleware is not installed on this project. We used it in the previous project to get the user's location. The weather should be based on the location of the hike so the ip address should not be necessary in this project